src/test/java/ca/wolfram/beta/symath/operations/MultiplyNodeTest.java

```diff
@@ -1,20 +1,31 @@
 package ca.wolfram.beta.symath.operations;
 
-import static org.junit.Assert.assertEquals;
-
-import org.junit.Test;
-
 import ca.wolfram.beta.symath.MathNode;
 import ca.wolfram.beta.symath.base.BaseNode;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
 
 public class MultiplyNodeTest {
     @Test
     public void string() {
-        MathNode multiply = MultiplyNode.create(BaseNode.create("x"), BaseNode.create("y"),BaseNode.create(1));
-        assertEquals("Multiply string should be concatenated values surrounded by brackets", "(x * y * 1)", multiply.toString());
+        MathNode multiply = MultiplyNode.create(BaseNode.create(2), BaseNode.create("x"), BaseNode.create("y"));
+        assertEquals("Multiply string should be concatenated values surrounded by brackets", "(x * y * 2)", multiply.toString());
```

Comment:
we should do something about the order of the output string. like having constants at the front.